zenoh/path.py

```diff
@@ -18,7 +18,11 @@ def __init__(self, path):
         self.__path_regex = re.compile('^[^?#*]+$')
         if not self.is_valid(path):
             raise ValueError("Invalid Path: {}".format(path))
-        self.path = path
+        self._path = path
```

Comment:
Sorry but, I just realised you changed the attribute name "path" into "_path".
As a consequence, you should rename it all across this file (i.e. each "self.path" => "selaf._path").

Reply:
@JEnoch done!